### PR TITLE
#395: Set up Code Climate

### LINKED ISSUE #395: Set up Code Climate
Set up Code Climate analysis for supported code, e.g. [ShellCheck](https://docs.codeclimate.com/docs/shellcheck).

### PR DESCRIPTION
**Pull request by alykhank**
Resolves #395.

### CODE REVIEW

File: script/uninstall.sh [hunk 1]
```diff
@@ -14,12 +14,8 @@ BIN_DIR="$PREFIX/bin"
 MAN_DIR="$PREFIX/share/man/man1"
 
 wait_for_user() {
-  if [ $(uname) = "Darwin" ]; then
-    read -n 1 -a CONTINUE -p "Press [y/N] to continue: " < /dev/tty
-  elif [ $(uname) = "Linux" ]; then
-    echo -n "Press [y/N] to continue: "
-    read CONTINUE < /dev/tty
-  fi
+  printf "Enter [y/N] to continue: "
```
Comment (adityatrivedi) [inline script/uninstall.sh]: Will these changes be made to the `gh-pages` branch too?

**Comment by adityatrivedi** [comment]
👍

File: script/uninstall.sh (hunk 1 above)
Comment (alykhank) [inline script/uninstall.sh]: Yes, and already tested on OS X and Linux.

**Comment by nehasingh2293** [comment]
👍

**Pull request merged**